bmc_remedy_itsm/help.md

```diff
@@ -1304,6 +1306,7 @@ _This plugin does not contain any custom output types._
 
 By default the API port is 8008.
 To configure the BMC server for API usage: https://docs.bmc.com/docs/ars9000/configuring-the-rest-api-529403222.html
```

Comment:
We might want to call out specifically the use of the REST api as many are still using the SOAP api.

Reply:
damn good call I completely forgot about the 2 different apis